int32 obj_index
float32 distance
geometry_msgs/Point32 point
